pkg/cmd/version/cmd.go

```diff
@@ -3,14 +3,21 @@ package version
 import (
 	"fmt"
 	"github.com/blang/semver"
+	"github.com/openshift/console-operator/pkg/console/version"
 	"github.com/spf13/cobra"
 	"strings"
+
+	cm "github.com/openshift/console-operator/pkg/console/subresource/configmap"
 )
 
 var (
-	Raw     = "v0.0.1"
-	Version = semver.MustParse(strings.TrimLeft(Raw, "v"))
-	String  = fmt.Sprintf("ConsoleOperator %s", Raw)
+	Raw        = "v0.0.1"
```

Comment:
Thanks for updating this!

I don't remember why there is a separate `pkg/console/version/version.go`.  Could be a relic of past versions of the operator.